testing/architecture-test/src/changes/accepted-changes/accepted-public-api-changes.json

```diff
@@ -48,6 +48,26 @@
                 "From non-null returning to null returning breaking change"
             ]
         },
+        {
+            "type": "org.gradle.api.GradleException",
+            "member": "Class org.gradle.api.GradleException",
+            "acceptation": "Making ResolutionProvider public API",
+            "changes": [
+                "org.gradle.api.ResolutionProvider"
+            ]
+        },
+        {
+            "type": "org.gradle.api.GradleException",
+            "member": "Constructor org.gradle.api.GradleException(java.lang.String,java.util.List)",
+            "acceptation": "Making ResolutionProvider public API",
+            "changes": []
+        },
+        {
+            "type": "org.gradle.api.GradleException",
+            "member": "Constructor org.gradle.api.GradleException(java.lang.String,java.util.List,java.lang.Throwable)",
+            "acceptation": "Making ResolutionProvider public API",
+            "changes": []
+        },
```

Comment:
Why are we getting these? I think we should be marking these constructors as `Incubating` + `@since` and these should go away